src/components/elementView/detailedView/assetView/assetView.tsx

```diff
@@ -94,28 +86,16 @@ export class AssetView extends React.PureComponent<
     toaster: (ref: Toaster) => (this.toaster = ref)
   };
 
-  public componentDidMount() {
```

Comment:
why this gone? because we're not calling getfields anymore?

Reply:
yes -> moved this logic to detailView, since its redundant to have it in assetView and modelView

Reply:
cool